css/portal-dashboard/custom-select.less

```diff
@@ -75,6 +72,14 @@
     transition-duration: .25s;
     pointer-events: none;
 
+    &.classNav {
+      width: 200px;
+    }
+
+    &.assignmentList {
+      width: 300px;
+    }
+
```

Comment:
I would make the width: 200px the default value and then just use width: 300px with a `wide` class to keep this component generic and unaware of its context (since it's designed to be a generic, reusable control).  So I would just do:
```
.list
  width: 200px;
 ...

    &.wide{
      width: 300px;
    }
...
```

Reply:
Then add the same to the `header` class.